.travis.yml

```diff
@@ -77,14 +83,19 @@ matrix:
     include:
         - otp_release: 20.0
           env: PRESET=dialyzer_only
+               SKIP_RELEASE=1 SKIP_BUILD_TESTS=1 SKIP_COV=1 SKIP_REPORT_UPLOAD=1
```

Comment:
`SKIP_CONFIGURE` could be added here as well. Moreover, configure makes sense if combined with a release. I suggest that we use only one var, for instance `SKIP_RELEASE`. What do you think?

Reply:
good idea, pushed.